core/embed/rust/src/ui/model_mercury/flow/confirm_output.rs

```diff
@@ -69,68 +113,302 @@ impl FlowState for ConfirmOutput {
+        .get(Qstr::MP_QSTR_summary_br_code)?
+        .try_into_option()?;
+
+    let cancel_text: Option<TString> = kwargs.get(Qstr::MP_QSTR_cancel_text)?.try_into_option()?;
+
+    // Address
+    let content_address = ConfirmBlobParams::new(TR::words__address.into(), address, None)
+        .with_subtitle(title)
+        .with_menu_button()
+        .with_footer(TR::instructions__swipe_up.into(), None)
+        .with_chunkify(chunkify)
+        .with_text_mono(text_mono)
+        .into_layout()?
+        .one_button_request(ButtonRequest::from_num(br_code, br_name));
+
+    // AddressMenu
+    let mut address_menu = VerticalMenu::empty();
+    let mut address_menu_items = Vec::<usize, 2>::new();
+    if account.is_some() && account_path.is_some() {
+        address_menu = address_menu.item(
+            theme::ICON_CHEVRON_RIGHT,
+            TR::address_details__account_info.into(),
+        );
+        unwrap!(address_menu_items.push(MENU_ITEM_ACCOUNT_INFO));
+    }
+    address_menu = address_menu.danger(
+        theme::ICON_CANCEL,
+        cancel_text.unwrap_or(TR::send__cancel_sign.into()),
+    );
+    unwrap!(address_menu_items.push(MENU_ITEM_CANCEL));
+    let content_address_menu = Frame::left_aligned(TString::empty(), address_menu)
+        .with_cancel_button()
+        .with_swipe(SwipeDirection::Right, SwipeSettings::immediate())
+        .map(move |msg| match msg {
+            FrameMsg::Content(VerticalMenuChoiceMsg::Selected(i)) => {
+                let selected_item = address_menu_items[i];
+                Some(FlowMsg::Choice(selected_item))
+            }
+            FrameMsg::Button(_) => Some(FlowMsg::Cancelled),
+        });
+
+    // AccountInfo
+    let ad = AddressDetails::new(TR::send__send_from.into(), account, account_path)?;
+    let content_account = ad.map(|_| Some(FlowMsg::Cancelled));
+
+    // AddressMenuCancel
+    let content_address_menu_cancel = Frame::left_aligned(
+        TR::send__cancel_sign.into(),
+        SwipeContent::new(PromptScreen::new_tap_to_cancel()),
+    )
+    .with_cancel_button()
+    .with_footer(TR::instructions__tap_to_confirm.into(), None)
+    .with_swipe(SwipeDirection::Down, SwipeSettings::default())
+    .with_swipe(SwipeDirection::Left, SwipeSettings::default())
+    .map(|msg| match msg {
+        FrameMsg::Content(PromptMsg::Confirmed) => Some(FlowMsg::Confirmed),
+        FrameMsg::Button(_) => Some(FlowMsg::Cancelled),
+        _ => None,
+    });
+
+    let res = if amount.is_some() {
+        let content_amount = ConfirmBlobParams::new(
+            TR::words__amount.into(),
+            amount.unwrap_or(Obj::const_none()),
+            None,
+        )
+        .with_subtitle(title)
+        .with_menu_button()
+        .with_footer(TR::instructions__swipe_up.into(), None)
+        .with_text_mono(text_mono)
+        .with_swipe_down()
+        .into_layout()?
+        .one_button_request(ButtonRequest::from_num(br_code, br_name));
```

Comment:
We should use `repeated_button_request` as that's the behavior from `T` and `R`.

Reply:
I see it fixed https://github.com/trezor/trezor-firmware/pull/4095/commits/26436ef6aa930f019e85acf0d0b48e22838b91b3